Stripe/STPPaymentMethodType.m

```diff
@@ -0,0 +1,305 @@
+//
+//  STPPaymentMethodType.m
+//  Stripe
+//
+//  Created by Brian Dorfman on 3/14/17.
+//  Copyright © 2017 Stripe, Inc. All rights reserved.
+//
+
+#import "STPPaymentMethodType.h"
+#import "STPPaymentMethodType+Private.h"
+
+#import "STPImageLibrary+Private.h"
+#import "STPImageLibrary.h"
+#import "STPLocalizationUtils.h"
+#import "STPPaymentMethod.h"
+
+/*
+ Reusable payment methods types cannot be used directly at payment time
+ (eg you cant pay with just "credit cards", it has to be a specific card.)
+
+ Single Use payment methods types are allowed to be payment methods types.
+ When the user attempts payment with one of these, we go create the source and
+ replace the payment method on the STPPaymentContext with the specific
+ created source
+ */
+
+#pragma mark - Reusable types
+
+@interface STPPaymentMethodTypeCreditCard : STPPaymentMethodType
+@end
+@implementation STPPaymentMethodTypeCreditCard
+
+- (UIImage *)paymentMethodImage {
+    return [STPImageLibrary addIcon];
+}
+
+- (UIImage *)paymentMethodTemplateImage {
+    return [STPImageLibrary addIcon];
+}
+
+- (NSString *)paymentMethodLabel {
+    return @"Credit Card";
```

Comment:
Hm, not sure – maybe we can mark them, but note that they don't need to be translated?